### ISSUE TITLE
`cem-plugin-expanded-types` breaks after 1.3.0 update

### ISSUE THREAD

**Issue by stefanpearson** [break-stuff/cem-tools#126]
Hey 👋 , I've encountered an issue after the 1.3.0 update:

```sh
> cem analyze --config ../../custom-elements-manifest.config.js

Error: Dynamic require of "path" is not supported
    at file:///Users/stefanp/Projects/design-system-pt/node_modules/cem-plugin-expanded-types/dist/index.js:12:9
    at src/main/options.js (file:///Users/stefanp/Projects/design-system-pt/node_modules/cem-plugin-expanded-types/dist/index.js:132339:21)
    at __require2 (file:///Users/stefanp/Projects/design-system-pt/node_modules/cem-plugin-expanded-types/dist/index.js:126384:56)
    at src/main/core.js (file:///Users/stefanp/Projects/design-system-pt/node_modules/cem-plugin-expanded-types/dist/index.js:133435:32)
    at __require2 (file:///Users/stefanp/Projects/design-system-pt/node_modules/cem-plugin-expanded-types/dist/index.js:126384:56)
    at ../../node_modules/prettier/index.js (file:///Users/stefanp/Projects/design-system-pt/node_modules/cem-plugin-expanded-types/dist/index.js:162797:16)
    at __require2 (file:///Users/stefanp/Projects/design-system-pt/node_modules/cem-plugin-expanded-types/dist/index.js:15:50)
    at file:///Users/stefanp/Projects/design-system-pt/node_modules/cem-plugin-expanded-types/dist/index.js:162874:31
    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:323:24)
[12:26:51] @custom-elements-manifest/analyzer: Created new manifest.
```

The issue occurs just by importing the module. I've rolled back to `1.2.0` for the time being and it works as expected.

Node: v20.12.2
NPM: 10.5.0

**Comment by break-stuff** [comment]
Thanks for letting me know!.let.me see if I can track down the issue.

**Comment by break-stuff** [comment]
It should be resolved in release 1.3.1. Let me know if you see any other issues.
